resources/resource_transformers/js/build.go

```diff
@@ -217,12 +220,24 @@ func toBuildOptions(opts Options) (buildOptions api.BuildOptions, err error) {
 		defines = cast.ToStringMapString(opts.Defines)
 	}
 
+	var sourceMap api.SourceMap
+	switch opts.SourceMap {
+	case "inline", "1":
```

Comment:
Please drop the magic numbers: Make "" = None. "inline"

Reply:
@bep This is my attempt to support true, false, None and "inline". Is there a better way to do it?
That way you can do
```hugo
    {{ $opts := dict "bundle" true "minify" false "sourcemap" Site.isServer "targetPath" $target "externals" $externals "defines" $defines }}
```
instead of
```hugo
    {{ $opts := dict "bundle" true "minify" false "sourcemap" (cond Site.isServer "inline" "") "targetPath" $target "externals" $externals "defines" $defines }}
```
But the second is not too much more difficult.

Reply:
This needs to be documented/explained to people in the future. It's better to have 1 way of doing it.